tests/test_raster.py

```diff
@@ -143,3 +147,52 @@ def test_raster_metadata_minmax_bad_params(
         ('var', 'var_2')
     )
     test_wsgi_app(raster_metadata, url, '400 Bad Request', None)
+
+
+@pytest.mark.online
+@pytest.mark.parametrize(
+    ("config", "environ"),
+    [
+        (
+            {
+                'root_url': 'http://tools.pacificclimate.org/dataportal/data/vic_gen1/',
+                'name': 'testing-server',
+                'version': 0,
+                'api_version': 0,
+                'handlers': [
+                    {
+                        'url': 'tasmin_day_BCSD+ANUSPLIN300+GFDL-ESM2G_historical+rcp26_r1i1p1_19500101-21001231.nc',
+                        'file': '/storage/data/climate/downscale/BCCAQ2/bccaqv2_with_metadata/tasmin_day_BCCAQv2+ANUSPLIN300_inmcm4_historical+rcp85_r1i1p1_19500101-21001231.nc'
+                    },
+                    {
+                        'url': 'pr+tasmin+tasmax+wind_day_CGCM3_A1B_run1_19500101-21001231.nc',
+                        'file': '/storage/data/projects/dataportal/data/vic_gen1_input/pr+tasmin+tasmax+wind_day_CGCM3_A1B_run1_19500101-21001231.nc'
+                    },
+                ],
+                'ensemble' : 'bccaq2',
+                'thredds_root' : 'http://docker-dev03.pcic.uvic.ca:30333/data'
+            },
+            {
+                'PATH_INFO' : 'tasmin_day_BCSD+ANUSPLIN300+GFDL-ESM2G_historical+rcp26_r1i1p1_19500101-21001231.nc.nc',
+                'QUERY_STRING' : 'tasmin[0:150][0:91][0:206]&',
+}
+        )
+    ]
+)
+def test_RasterServer_orca(mm_database_dsn, config, environ):
+    r_server = RasterServer(mm_database_dsn, config)
+    resp = r_server(environ, Response())
+
+    assert resp.status_code == 301
```

Comment:
We may want to consider making a test where a bad request is made and we check what `status_code` is sent back.